.circleci/config.yml

```diff
@@ -163,40 +245,66 @@ jobs:
       - deploy:
           name: Deploy fala to production
           command: |
-            .circleci/deploy_to_kubernetes production ${ECR_DOCKER_REPO_BASE}-webapp ${ECR_DOCKER_REPO_BASE}-nginx
+            .circleci/deploy_to_kubernetes production ${ECR_DOCKER_REPO_BASE}
       - deploy:
           name: Notify Slack channel
-          command: .circleci/notify_slack_channel production
+          command: .circleci/notify_slack_channel production-live-1
 
 workflows:
   version: 2
   build_test_and_deploy:
     jobs:
-      - lint-web
-      - test-web
-      - build_web:
+      - lint
+      - test
+      - live-0_build_web:
           requires:
-            - test-web
-            - lint-web
-      - build_nginx:
+            - lint
+            - test
+      - live-0_build_nginx:
           requires:
-            - test-web
-      - staging_deploy_approval:
+            - lint
+            - test
+      - live-0_staging_deploy_approval:
           type: approval
           requires:
-            - build_web
-            - build_nginx
-      - staging_deploy:
+            - live-0_build_web
+            - live-0_build_nginx
+      - live-0_staging_deploy:
           requires:
-            - staging_deploy_approval
-      - production_deploy_approval:
+            - live-0_staging_deploy_approval
+      - live-0_production_deploy_approval:
           type: approval
           requires:
-            - staging_deploy
+            - live-0_staging_deploy
           filters:
             branches:
               only:
                 - master
-      - production_deploy:
+      - live-0_production_deploy:
+          requires:
+            - live-0_production_deploy_approval
+      - live-1_build:
+          requires:
+            - lint
+            - test
+          context: laa-fala-live-1
+      - live-1_staging_deploy_approval:
+          type: approval
+          requires:
+            - live-1_build
+      - live-1_staging_deploy:
           requires:
-            - production_deploy_approval
+            - live-1_staging_deploy_approval
+          context: laa-fala-live-1
+      # - live-1_production_deploy_approval:
```

Comment:
It looks like the `live-1_production_deploy` job would not work just now.
What do you think of removing all related code until it works?

Reply:
Not particularly bothered either way